src/components/AppLayout/AppHeader.tsx

```diff
@@ -562,13 +562,17 @@ export function AppHeader({
     ],
     [
       currentUser,
+      currentUser?.username,
       features.imageTrainingResults,
       features.bounties,
       features.buzz,
       features.questions,
+      features.clubs,
+      features.vault,
       bookmarkedModelsCollection,
       bookmarkedArticlesCollection,
       router.asPath,
+      theme,
```

Comment:
I don't remember why, but we were omitting having `theme` in the deps array for a reason 😬 can you confirm everything works as expected here?

Reply:
Oh. I'll comment it out just to be safe.